### ISSUE TITLE
Start redis docker image with sysctl settings

### ISSUE THREAD

**Issue by MikeSchroll** [dokku/dokku-redis#74]
Redis by default outputs some sysctl errors:

```
1:S 21 Apr 03:01:51.589 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
1:S 21 Apr 03:01:51.589 # Server started, Redis version 3.2.5
1:S 21 Apr 03:01:51.589 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
1:S 21 Apr 03:01:51.589 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
```

At least one of these, somaxconn, can be solved by taking advantage of the [new --sysctl switch](https://docs.docker.com/engine/reference/commandline/run/)

Could it be considered to incorporate this?

**Comment by josegonzalez** [comment]
Sure, pull requests welcome.

**Comment by ambethia** [comment]
Any luck with this? I'm having memory issues with redis under dokku, I believe being able to set these recommended values will resolve it.

**Comment by josegonzalez** [comment]
@ambethia 

> Sure, pull requests welcome.

**Comment by josegonzalez** [comment]
Closed by #108.
